### PR TITLE
Styling review request & PR

### PR DESCRIPTION
**Pull request by Pandry**
Referring to issue #51

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,6 +4,74 @@ Watch the introduction video [video \(IT\)](https://www.youtube.com/watch?v=KH8x
...
+5. A blank line
+6. Other libraries' .h files.
+7. Your project's .h files.
+
+access level order:
+- public
+- protected
+- private
+
+### How to send an issue
```
Comment (Pandry) [inline CONTRIBUTING.md:70]: Should this be a thing?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,6 +4,74 @@ Watch the introduction video [video \(IT\)](https://www.youtube.com/watch?v=KH8x
...
+
+1. headerfiles
+2. A blank line
+3. C system files.
+4. C++ system files.
+5. A blank line
+6. Other libraries' .h files.
+7. Your project's .h files.
+
+access level order:
```
Comment (Pandry) [inline CONTRIBUTING.md:65]: More specific?
Also, should a number list be used instead?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -4,6 +4,74 @@ Watch the introduction video [video \(IT\)](https://www.youtube.com/watch?v=KH8x
...
+
+@param nameOfTheFirstParameter Description of the first parameter
+@param nameOfTheNParameter Description of the n parameter
+@returns What the function returns
+@note Other notes about parameter or the function itself
+*/
+void nameOfTheFunction(nameOfTheFirstParameter, nameOfTheNParameter){}
+``` 
+
+Order of declaration:
```
Comment (Pandry) [inline CONTRIBUTING.md:47]: Is this clear enough?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -4,6 +4,74 @@ Watch the introduction video [video \(IT\)](https://www.youtube.com/watch?v=KH8x
...
+- Constants written in SNAKE_CASE
+- Other use camelCase
+
+- Public functions/objects/... -> FirstLetterUppercase
+- Non-public functions/objects/... -> firstLetterLowercase
+
+- Comment before function:
+``` 
+/*
+nameOfTheFunction Long description of the function's function
```
Comment (Pandry) [inline CONTRIBUTING.md:37]: Should I also write a class example?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -4,6 +4,74 @@ Watch the introduction video [video \(IT\)](https://www.youtube.com/watch?v=KH8x
...
+- Use inline function for 8 lines long functions or less
+- Do not define implicit conversions. Use the explicit keyword
+- Use a struct only for passive objects that carry data; everything else is a class.
+- Declare variables in a local way whenever is possible (best prefix for a global variable is `//`)
+- Initialize variables when declaring them
+- Align the equal sign when declaring multiple variables
+- readability > functionality, don't be afraid of writing 5 lines of code more
+- Use namespaces, the name should be about the functionality itself
+- **DO NOT** use `using` (e.g. `using namespace std;`)
+- At the end of namespace, comment it's end `} // namespace nsName`
```
Comment (Pandry) [inline CONTRIBUTING.md:24]: Unclear

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -4,6 +4,74 @@ Watch the introduction video [video \(IT\)](https://www.youtube.com/watch?v=KH8x
...
+- Prefer small and focused functions
+- Use inline function for 8 lines long functions or less
+- Do not define implicit conversions. Use the explicit keyword
+- Use a struct only for passive objects that carry data; everything else is a class.
+- Declare variables in a local way whenever is possible (best prefix for a global variable is `//`)
+- Initialize variables when declaring them
+- Align the equal sign when declaring multiple variables
+- readability > functionality, don't be afraid of writing 5 lines of code more
+- Use namespaces, the name should be about the functionality itself
+- **DO NOT** use `using` (e.g. `using namespace std;`)
```
Comment (Pandry) [inline CONTRIBUTING.md:23]: `using` is too generic

File: CONTRIBUTING.md (hunk 1 above)
Comment (AndreaIdini) [reply CONTRIBUTING.md:70]: Is there, is fine.

File: CONTRIBUTING.md (hunk 5 above)
Comment (AndreaIdini) [reply CONTRIBUTING.md:24]: "its", but you can leave just "comment".

File: CONTRIBUTING.md (hunk 6 above)
Comment (AndreaIdini) [reply CONTRIBUTING.md:23]: "`using` namespaces"

**Review by AndreaIdini — APPROVED** [review]
ok!

**Pull request merged**